sass/modules/_resets.scss

```diff
@@ -4,6 +4,8 @@
   outline: 0;
   text-decoration: none;
   box-sizing: border-box;
+  font-family: lato,helvetica,arial,sans-serif;
```

Comment:
"Lato", Helvetica, Arial, sans-serif and variants thereof are defined in many places.  Can they please use a common variable in sass/_variables.scss to make it easier to update?